source/dubregistry/viewutils.d

```diff
@@ -77,9 +77,9 @@ string formatFuzzyDate(SysTime st)
 	else return format("%s years ago", now.year - st.year);
 }
 
-string formatScore(string rootDir, float score)
+string formatScore(float score)
 {
-	return format(`<a href="%sdevelop#package-scoring">%.1f</a>`, rootDir, score);
+	return format(`<a href="https://dub.pm/develop#package-scoring">%.1f</a>`, score);
```

Comment:
This is change is wrong. A website content should always be relative and never hard code the domain name.

Reply:
The documentation is now on dub.pm, whereas the main registry is at code.dlang.org, so this is actually a cross-domain link.

Reply:
Ah yes, thanks for clarifying that. What I meant is that hard coding the docs domain looks wrong. I think the correct way approach is to read the domain name from a configuration.